addons/spreadsheet_account/models/account.py

```diff
@@ -108,14 +163,60 @@ def spreadsheet_fetch_debit_credit(self, args_list):
             include_unposted: bool
         }]
         """
+        all_accounts, all_lines = self._get_accounts_and_lines_for_all_cells(
+            args_list,
+            extra_aggregates=['partner_id:array_agg', 'debit:array_agg', 'credit:array_agg']
```

Comment:
Doesn't using `array_agg` almost come to the same as fetching all the recordsets? (in big O notation at least)
Seems like it is defeating the purpose of `_read_group`


What would make sense is to group by `partner_id` instead and take the `sum` of `debit`/`credit`/...